pkg/subctl/cmd/gather.go

```diff
@@ -82,23 +85,43 @@ func gatherData() {
 	err := checkGatherArguments()
 	exitOnError("Invalid arguments", err)
 
-	for module, ok := range gatherModuleFlags {
-		if ok {
-			for dataType, ok := range gatherTypeFlags {
-				if ok {
-					status := cli.NewStatus()
-					status.Start(fmt.Sprintf("Gathering %s %s...", module, dataType))
-					status.End(cli.CheckForError(gatherFuncs[module](status, dataType)))
+	configs, err := getMultipleRestConfigs(kubeConfig, kubeContext)
+	exitOnError("Error getting REST config", err)
+	for _, item := range configs {
+		clientSet, err := kubernetes.NewForConfig(item.config)
+		exitOnError("error getting clientset", err)
+		clustername := item.clusterName
+		fmt.Println("Gathering information from", clustername)
```

Comment:
Println was to print an empty line in between each cluster output. Like this:
`Gathering information from cluster1
Logs collected

Gathering information from cluster2
Logs collected`

But since, I am removing logic to collect logs from all clusters but just 1, this is removed.

Reply:
Either way we should use the `Status` for all output

Reply:
I think it's fine to leave, see at the example output where there is a prefix "Gathering information from ..." before all the tick statuses. So there's a separation between the cluster items.

Reply:
yeah good point.